.azure-pipelines/testcases_collection/testcases.collection.yml

```diff
@@ -0,0 +1,97 @@
+              https://download.docker.com/linux/ubuntu \
+              $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+            sudo apt-get -o DPkg::Lock::Timeout=600 update
+            sudo apt-get -o DPkg::Lock::Timeout=600 install docker-ce docker-ce-cli containerd.io -y
+          displayName: 'Install Docker'
+
+        - script: |
+            set -x
+
+            sudo docker pull sonicdev-microsoft.azurecr.io:443/docker-sonic-mgmt:latest
+            sudo docker rm -f sonic-mgmt-collect || true
+            sudo docker run --rm -dt --name sonic-mgmt-collect \
+              -v $(System.DefaultWorkingDirectory)/sonic-mgmt-int:/var/src/sonic-mgmt-int \
+              -v $(System.DefaultWorkingDirectory)/sonic-mgmt:/var/src/sonic-mgmt \
+              sonicdev-microsoft.azurecr.io:443/docker-sonic-mgmt:latest \
+              /bin/bash
+          displayName: 'Prepare sonic-mgmt docker container'
+
+        - task: AzureCLI@2
+          displayName: Testcases Collection for sonic-mgmt-int
+          inputs:
+            azureSubscription: "SONiC-Automation"
+            scriptType: 'bash'
+            scriptLocation: 'inlineScript'
+            inlineScript: |
+              set -x
+
+              accessToken=$(az account get-access-token --resource https://api.kusto.windows.net --query accessToken -o tsv)
+              export ACCESS_TOKEN=$accessToken
+
+              sudo docker exec -e ACCESS_TOKEN=$accessToken -e TEST_REPORT_INGEST_KUSTO_CLUSTER_URL=$(TEST_REPORT_INGEST_KUSTO_CLUSTER_BACKUP) -t -w /var/src/sonic-mgmt-int/tests sonic-mgmt-collect python ../.azure-pipelines/testcases_collection/collect_testcases.py --location . --db_name SonicTestData --db_table $(TESTCASES_TABLE) --db_table_mapping $(TESTCASES_TABLE_MAPPING) --repo_url $(SONIC_MGMT_INT_URL) --branch $(Build.SourceBranchName)
+          env:
+            TEST_REPORT_INGEST_KUSTO_CLUSTER_URL: $(TEST_REPORT_INGEST_KUSTO_CLUSTER_BACKUP)
+
+        - task: AzureCLI@2
+          displayName: Testcases Collection for sonic-mgmt
+          inputs:
+            azureSubscription: "SONiC-Automation"
+            scriptType: 'bash'
+            scriptLocation: 'inlineScript'
+            inlineScript: |
+              set -x
+
+              accessToken=$(az account get-access-token --resource https://api.kusto.windows.net --query accessToken -o tsv)
+              export ACCESS_TOKEN=$accessToken
+
+              sudo docker exec -e ACCESS_TOKEN=$accessToken -e TEST_REPORT_INGEST_KUSTO_CLUSTER_URL=$(TEST_REPORT_INGEST_KUSTO_CLUSTER_BACKUP) -t -w /var/src/sonic-mgmt/tests sonic-mgmt-collect python ../.azure-pipelines/testcases_collection/collect_testcases.py --location . --db_name SonicTestData --db_table $(TESTCASES_TABLE) --db_table_mapping $(TESTCASES_TABLE_MAPPING) --repo_url $(SONIC_MGMT_URL) --branch $(resources.repositories.sonic-mgmt.ref)
```

Comment:
Why still pass down BACKUP URL here as argument?
-e TEST_REPORT_INGEST_KUSTO_CLUSTER_URL=$(TEST_REPORT_INGEST_KUSTO_CLUSTER_BACKUP)

The value of BACKUP already set to `TEST_REPORT_INGEST_KUSTO_CLUSTER_URL`, yes?

Reply:
The env is to pass the environment variable to the agent, and we need to execute our command in docker, so this is to pass the environment variable into docker.

Reply:
In line 97, value of `$(TEST_REPORT_INGEST_KUSTO_CLUSTER_BACKUP)` is assigned to environment variable `TEST_REPORT_INGEST_KUSTO_CLUSTER_URL`, then in this script block, you can always refer to environment variable `TEST_REPORT_INGEST_KUSTO_CLUSTER_URL`.

If directly use `$(TEST_REPORT_INGEST_KUSTO_CLUSTER_BACKUP)` in script part, then line 97 is not used.

Reply:
Removed the `env` in both tasks